src/pfb/etl/etl.py

```diff
@@ -49,7 +49,7 @@ def extract(self):
         with open(self.pfbfile, "rb") as fo:
             avro_reader = reader(fo)
             # skip schema
-            avro_reader.next()
+            next(avro_reader)
```

Comment:
i'm curious what's the difference between the 2? does this fix a bug?

Reply:
yes, the `avro_reader.next()` says that no member next exists. That's why we have to change to `next(avro_reader)